### DISCUSSION TITLE
Bentely Login API

### DISCUSSION THREAD

**Discussion by Zubair-KPM** [iTwin/community#29] [Q&A] [answered]
Hi, Everyone 

Can anyone tell that how i can get the Bentely login API and the whole process?

**Comment by palvindersingh** [comment]
Was there any reply to this question?

**Reply by Zubair-KPM** [reply]
#aruniverse has given the API Authentication link but i need proper example to understand it better way

**Comment by aruniverse** [comment, marked as answer]
Have you seen https://developer.bentley.com/apis/overview/authorization/ ?

**Reply by Zubair-KPM** [reply]
Hi, actually i need an example to complete this task so, if you have any example code than let me know

**Reply by aruniverse** [reply]
Have you tried using the auth-clients packages from https://github.com/iTwin/auth-clients

**Reply by Zubair-KPM** [reply]
no i haven't used it, Now i will try 

Thanks
